odoo/http.py

```diff
@@ -2375,7 +2375,9 @@ def fake_start_response(status, headers):
                         _logger.warning("Database or registry unusable, trying without", exc_info=e.__cause__)
                         request.db = None
                         request.session.logout()
-                        if httprequest.path in ('/web', '/web/login', '/test_http/ensure_db'):
+                        if (httprequest.path in ('/web', '/web/login', '/test_http/ensure_db')
+                            or httprequest.path.startswith('/odoo')
```

Comment:
Are you sure you don't prefer to handle `/odoo` and `/odoo/*` separately to avoid matching stuff like `/odoorn/stadplan` ?

Reply:
indeed a better idea, thanks!